Sources/Confidence/ConfidenceClientOptions.swift

```diff
@@ -4,15 +4,18 @@ struct ConfidenceClientOptions {
     public var credentials: ConfidenceClientCredentials
     public var region: ConfidenceRegion
     public var initializationStrategy: InitializationStrategy
+    public var timeoutIntervalForRequest: Double
 
     public init(
         credentials: ConfidenceClientCredentials,
         region: ConfidenceRegion? = nil,
-        initializationStrategy: InitializationStrategy = .fetchAndActivate
+        initializationStrategy: InitializationStrategy = .fetchAndActivate,
+        timeoutIntervalForRequest: Double = 0
```

Comment:
what does 0 mean here?

Reply:
right, i should remove this. i set this up so that the timeout would init with a null and get overwritten, but we don't set any before we actually create a networkclient in confidence builder